src/widget/OktaSignIn.js

```diff
@@ -80,15 +80,38 @@ var OktaSignIn = (function () {
      * @param success - success callback function
      * @param error - error callback function
      */
+    var router;
     function render(options, success, error) {
-      var router = new LoginRouter(_.extend({}, config, options, {
+      if (router) {
+        throw 'Cannot run two instances of the widget simultaneously. Call remove() first.';
```

Comment:
nit, maybe word this like:

An instance of the widget has already been rendered. Call remove() first.